### PR TITLE
Jvenstad/deporch cleanup

### PR DESCRIPTION
**Pull request by jonmv**
@mpolden please review.

Just elimination of dead code, formatting and that helper you requested 😄

### CODE REVIEW

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java [hunk 1]
```diff
@@ -539,17 +539,8 @@ public void restart(DeploymentId deploymentId, Optional<Hostname> hostname) {
...
-    /** Deactivate a known deployment of the given application */
-    public void deactivate(Application application, Deployment deployment, boolean requireThatDeploymentHasExpired) {
-        deactivate(application, deployment.zone(), Optional.of(deployment), requireThatDeploymentHasExpired);
-    }
-
-    private void deactivate(Application application, ZoneId zone, Optional<Deployment> deployment,
-                            boolean requireThatDeploymentHasExpired) {
-        if (requireThatDeploymentHasExpired && deployment.isPresent()
+        Optional<Deployment> deployment = Optional.empty();
+        if (false && deployment.isPresent()
```
Comment (mpolden) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java]: `if (false`?

**Review by mpolden — CHANGES_REQUESTED** [review]

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/JobStatus.java [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import com.yahoo.component.Version;
 import com.yahoo.vespa.hosted.controller.Controller;
 import com.yahoo.vespa.hosted.controller.deployment.DeploymentTrigger;
+import org.jetbrains.annotations.TestOnly;
```
Comment (mpolden) [inline controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/JobStatus.java]: Probably unintentional..

File: controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java (hunk 1 above)
Comment (jonmv) [reply controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java]: Huh? I clearly remember deleting those lines of code >_<

**Comment by jonmv** [comment]
PTAL.

**Review by mpolden — APPROVED** [review]

**Pull request merged**